tests/Microsoft.Identity.Test.Integration.netcore/HeadlessTests/GenericAuthorityTests.cs

```diff
@@ -45,6 +59,36 @@ public async Task ShouldSupportClientCredentialsWithDuendeDemoInstanceAsync()
             Assert.AreEqual(TokenSource.Cache, response2.AuthenticationResultMetadata.TokenSource);
         }
 
+        /// Based on the publicly available https://demo.duendesoftware.com/
+        [TestMethod]
+        public async Task ShouldSupportAcquireTokenInteractiveWithDuendeDemoInstanceAsync()
```

Comment:
This is a great test! Thank you!

The only comment I have is to move it to the `InteractiveFlowTests` class, and use the attribute `        `[RunOn(TargetFrameworks.NetCore)]` since we want to run this on .NET core only, not on .NET fwk (just because UI tests are slow and our build is already super slow).